### PR TITLE
Replace Gaurav's profile image

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

**Comment by gsheni** [comment]
@g12mcgov Ready for review

**Comment by g12mcgov** [comment]
@gsheni can you make it webp format? I'll then merge after.

It'll compress better + load faster.

File: src/components/WeddingParty.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const groomsmen = [
   },
   {
     name: "Gaurav Sheni",
-    image: "/images/groomsmen/gaurav.webp",
+    image: "/images/groomsmen/gaurav.jpg",
```
Comment (gsheni) [inline src/components/WeddingParty.tsx:16]:
```suggestion
    image: "/images/groomsmen/gaurav.webp",
```

**Comment by gsheni** [comment]
@g12mcgov done.

**Pull request merged**